examples/demo/updating_plot/updating_plot2.py

```diff
@@ -69,13 +74,13 @@ def timer_tick(self):
         self.current_index += self.increment
         if self.current_index > self.numpoints:
             self.current_index = self.numpoints
-        self.plot.index.set_data(self.x_values[: self.current_index])
-        self.plot.value.set_data(self.y_values[: self.current_index])
+        self.plot.index.set_data(self.x_values[: int(self.current_index)])
```

Comment:
ah you are right, I actually made this as the fix first when I first encountered the issue, but then realized it could be fixed at the source.  I can undo this change